libs/domain/search/src/services/adapter/default-suggestion.adapter.ts

```diff
@@ -23,20 +23,33 @@ export class DefaultSuggestionAdapter implements SuggestionAdapter {
     return query ?? '';
   }
 
-  get({ query }: SuggestionQualifier): Observable<Suggestion> {
-    const include = [
-      ApiProductModel.Includes.AbstractProducts,
-      ApiProductModel.Includes.ConcreteProducts,
-      ApiProductModel.Includes.ConcreteProductImageSets,
-      ApiProductModel.Includes.ConcreteProductPrices,
-      ApiProductModel.Includes.ConcreteProductAvailabilities,
-      ApiProductModel.Includes.Labels,
-    ].join(',');
-
-    return this.http
-      .get<ApiSuggestionModel.Response>(
-        `${this.SCOS_BASE_URL}/${this.queryEndpoint}?q=${query}&include=${include}`
+  get({ query, entries }: SuggestionQualifier): Observable<SuggestionResponse> {
+    if (
+      !entries ||
+      entries.some((entry) =>
+        [
+          SuggestionField.Categories,
+          SuggestionField.Completion,
+          SuggestionField.Products,
+        ].includes(entry as SuggestionField)
       )
-      .pipe(this.transformer.do(SuggestionNormalizer));
+    ) {
+      const include = [
```

Comment:
can we optimize this one, if no products are needed, we don't have to include them?